Author: Chris Lamb <chris@chris-lamb.co.uk>
Description: Fix Bashisms in fig2ps2tex /bin/sh script

--- a/fig2dev/fig2ps2tex.sh.script
+++ b/fig2dev/fig2ps2tex.sh.script
@@ -14,11 +14,11 @@
 xsp=`echo "scale=3; ( $bbox4 - $bbox2 ) / 72" | bc`
 ysp=`echo "scale=3; ( $bbox5 - $bbox3 ) / 72" | bc`
 
-echo "\makebox[${xsp}in][l]{"
-echo "  \vbox to ${ysp}in{"
-echo "    \vfill"
-echo "    \special{psfile=$1}"
-echo "  }"
-echo "  \vspace{-\baselineskip}"
-echo "}"
+printf "\makebox[${xsp}in][l]{\n"
+printf "  \\\vbox to ${ysp}in{\n"
+printf "    \\\vfill\n"
+printf "    \special{psfile=$1}\n"
+printf "  }\n"
+printf "  \\\vspace{-\\\baselineskip}\n"
+printf "}\n"
 
